### PR TITLE
8 patch /api/articles/article

### PR DESCRIPTION
**Pull request by Jord000**
patch request to add or reduce votes with testing and error handling

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -234,23 +233,97 @@ describe('POST /api/articles/:article_id/comments', () => {
...
+      .then(({ body }) => {
+        expect(body.msg).toBe('Bad Request');
+      });
+  });
+});
+
+describe('PATCH: /api/articles/:article_id', () => {
+  const incVotes = { inc_votes: 20 };
+  const wrongVote = { notcorrect: 'incorrect' };
+  const reduceVotes = { inc_votes: -200 };
```
Comment (johnhutcheon) [inline __tests__/app.test.js:260]: Nice ordering at the top of this describe block 🙌

**Review by Keb84 — APPROVED** [review]
Really nice work. merge when ready 👍

File: __tests__/app.test.js [hunk 2]
```diff
@@ -234,23 +233,97 @@ describe('POST /api/articles/:article_id/comments', () => {
...
+            title: 'UNCOVERED: catspiracy to bring down democracy',
+            topic: 'cats',
+            author: 'rogersop',
+            body: 'Bastet walks amongst us, and the cats are taking arms!',
+            created_at: '2020-08-03T13:14:00.000Z',
+            article_img_url:
+              'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
+            votes: -200,
+          });
+        }
```
Comment (johnhutcheon) [inline __tests__/app.test.js:326]: Could you just add one more test for if we send a request to a valid but non-existent path please?

**Review by johnhutcheon — CHANGES_REQUESTED** [review]
Really nice work again Jordan. Just one test missing on this one. Take a look at my comment and add, then resubmit please. Thanks! 🙂

**Review by nickkagame — APPROVED** [review]
good work!

**Pull request merged**